lib/buildConnectorsJson/prettyTitle.js

```diff
@@ -7,6 +7,20 @@
 var sentenceCase = require('mout/string/sentenceCase');
 var unCamelCase  = require('mout/string/unCamelCase');
 
+let matchWord = (word) => { return new RegExp(`\\b${word}\\b`, 'gi'); };
+
+const handleNameEdgeCases = (name) => {
+	const IdRegex = matchWord('id');
+	const IdsRegex = matchWord('ids');
+	const UrlRegex = matchWord('url');
```

Comment:
Is there a need for these regexes to be reinitialised inside the function every time? If not, move them up/out of the function